gadgetapi.php

```diff
@@ -14,6 +14,7 @@
 //Expand text from postvars
 $page = new Page();
 $page->text = $originalText;
+$_REQUEST["slow"] = 1;
```

Comment:
You can't set $_REQUEST variables, unfortunately.  
To do this, visit citationbot.com/rest-of-url.html?**slow=1**